src/lib/eap_aka_sim/state_machine.c

```diff
@@ -722,7 +722,7 @@ RESUME(store_pseudonym)
 
 			fr_value_box_bstrdup_buffer_shallow(NULL, &vp->data, NULL, identity, false);
 		}
-		pair_update_request(&new, attr_session_id);
+		MEM(pair_update_request(&new, attr_session_id) >= 0);
```

Comment:
@arr2036 maybe we could use the _Coccinelle_ to detect the use of `pair_update_request()` without such `MEM()`. what do you think @arr2036 ?

Reply:
Looking more closely at main_config.c, it ought to return -1 if the `fr_value_box_bstr_realloc()` fails in `xlat_config_escape()`. I will make that change.

Reply:
Yeah for these functions we could add an assert.  We'd need to go through and remove all the MEM() wrappers around the calls though.  If you want to look at it @jpereira go ahead.